dashboard/src/components/layout/AppLayout.tsx

```diff
@@ -547,7 +556,19 @@ const AppLayout: React.FC<Props> = ({
               </div>
             </div>
           </div>
-
+          {showAlert && (
+            <Alert className="flex rounded-none flex-col items-center bg-primary/90 text-white justify-center">
+              <AlertTitle className="text-xl flex items-center text-white gap-4 justify-center">
+                Waiting for your application to start
+                <Spinner color="info" className="mb-0.5" />
+              </AlertTitle>
+              <AlertDescription className="text-lg text-center">
+                {!data
+                  ? "Ensure Nitric is running by executing `nitric start` and start your application using its appropriate command"
+                  : "Ensure your application is running by executing its appropriate command"}
```

Comment:
```suggestion
                  ? "Dashboard disconnected from nitric server, ensure nitric is running by executing `nitric start`."
                  : "Nitric is running but hasn't received a connection from your application, ensure your application is running."}
```

I wanted to change the message so it didn't refer to 'appropriate command' since other options are available, like running an app in an IDE debugger for example.